messages/ru.json

```
@@ -4,30 +4,30 @@
   "address-form.optional": "Необязательно",
   "address-form.edit": "Редактировать",
   "address-form.search": "Найти",
-  "address-form.geolocation.example.ARG": "Например: Av. del Libertador 1473, Buenos Aires",
-  "address-form.geolocation.example.BOL": "Например: Av Arce, 2556, La Paz",
-  "address-form.geolocation.example.BRA": "Например: Av Paulista, 1578, São Paulo",
-  "address-form.geolocation.example.CAN": "Например: 450 Wilbrod St, Ottawa",
-  "address-form.geolocation.example.CHL": "Например: Los Militares, 6191, Santiago",
-  "address-form.geolocation.example.COL": "Например: Calle 93 # 14-20, Bogotá",
-  "address-form.geolocation.example.ECU": "Например: Av Rio Amazonas, N 37-61, Quito",
-  "address-form.geolocation.example.ESP": "Например: Calle Fernando el Santo, 6, Madrid",
-  "address-form.geolocation.example.GTM": "Например: 6A Calle 6, Guatemala",
-  "address-form.geolocation.example.MEX": "Например: Calle de Tacuba 8, Ciudad de México",
+  "address-form.geolocation.example.ARG": "Eg: Av. del Libertador 1473, Buenos Aires",
+  "address-form.geolocation.example.BOL": "Eg: Например: Av Arce, 2556, La Paz",
```

shouldn't this "Например" be removed?
```suggestion
  "address-form.geolocation.example.BOL": "Eg: Av Arce, 2556, La Paz",
```